src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -77,13 +77,21 @@ public async Task<string> GetMarkdownValidationRules((string repositoryUrl, stri
 
         public async Task<string> GetAllowlists((string repositoryUrl, string branch) tuple)
         {
-            return await FetchValidationRules($"/route/validationmgt/validation/allowlists", tuple.repositoryUrl, tuple.branch);
+            return await FetchValidationRules(
+                   $"/route/taxonomyservice/taxonomies/simplified?" +
+                   $"name=ms.author&name=ms.devlang&name=ms.prod&name=ms.service&name=ms.topic&name=devlang&name=product",
+                   tuple.repositoryUrl,
+                   tuple.branch);
         }
 
         public async Task<string> GetMetadataSchema((string repositoryUrl, string branch) tuple)
         {
             var metadataRules = FetchValidationRules($"/route/validationmgt/rulesets/metadatarules", tuple.repositoryUrl, tuple.branch);
-            var allowlists = FetchValidationRules($"/route/validationmgt/validation/allowlists", tuple.repositoryUrl, tuple.branch);
+            var allowlists = FetchValidationRules(
+                $"/route/taxonomyservice/taxonomies/simplified?" +
```

Comment:
taxonomy service api is public, we don't need use the OPS routing API, and we did not configure it yet, that the reason they are so many difference.

Reply:
OK, changed it.